src/platform/forms/save-in-progress/SaveInProgressIntro.jsx

```diff
@@ -209,14 +208,15 @@ class SaveInProgressIntro extends React.Component {
           {unauthStartButton}
           {!this.props.hideUnauthedStartLink && (
             <p>
-              <button
-                className="va-button-link schemaform-start-button"
+              <Link
                 onClick={this.goToBeginning}
```

Comment:
With the change to `<Link>`, it looks like the `router.push` is [already being handled](https://github.com/remix-run/react-router/blob/850de933444d260bfc5460135d308f9d74b52c97/modules/Link.js#L95). Do we still need to manually be doing that?

https://github.com/department-of-veterans-affairs/vets-website/blob/61fec575eb0940059e0b56efd93bfd41c9851b00/src/platform/forms/save-in-progress/SaveInProgressIntro.jsx#L302-L305

Reply:
Yeah - I wasn't sure if I should leave that in or not either. I suppose if we got rid of that, should probably rename this to `recordNoLoginStartEvent` or something. What do you think?

Reply:
The `handleClick` that you've got now seems fine to me.